examples/acceptor/src/main/java/com/paritytrading/philadelphia/acceptor/Session.java

```diff
@@ -77,7 +77,7 @@ public void reject(FIXConnection connection, FIXMessage message) throws IOExcept
             public void logon(FIXConnection connection, FIXMessage message) throws IOException {
                 connection.sendLogon(true);
 
-                connection.updateCompID(report);
+                connection.reprepare(report);
```

Comment:
This definitely reads like a typo 😆

Reply:
I think I got the feedback I needed here. 😂

Reply:
`resetOrPrepare`? Or avoiding it with a `setOrAdd` behavior in `FIXMessage`?